src/cls/rgw_gc/cls_rgw_gc_client.cc

```diff
@@ -0,0 +1,103 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#include <errno.h>
+
+#include "cls/rgw/cls_rgw_ops.h"
+#include "cls/rgw_gc/cls_rgw_gc_ops.h"
+#include "cls/queue/cls_queue_ops.h"
+#include "cls/rgw_gc/cls_rgw_gc_const.h"
+#include "cls/queue/cls_queue_const.h"
+#include "cls/rgw_gc/cls_rgw_gc_client.h"
+
+using namespace librados;
+
+void cls_rgw_gc_queue_init(ObjectWriteOperation& op, const string& queue_name, uint64_t size, uint64_t num_urgent_data_entries)
```

Comment:
what is the `queue_name` used for?

Reply:
It was for debugging purposes, will remove it